pkgs/development/python-modules/pyxlsb/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "pyxlsb";
+  version = "1.0.10";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-gGLR6oYm0/GYDosc/pGkSDdHRJJC7LYQE7wt+FQ19oU=";
+  };
+
```

Comment:
Can we add:

```
# package has no tests
doCheck = false;
```

explicitly? `doCheck` is true by default, and `setuptoolsCheckPhase` will run. It won't do anything, but IMO we might as well skip it and document what's going on at the same time.